src/Javis.jl

```diff
@@ -184,7 +186,10 @@ Renders all previously defined [`Object`](@ref) drawings to the user-defined `Vi
 - `tempdirectory::String`: The folder where each frame is stored
     Defaults to a temporary directory when not set
 - `liveview::Bool`: Causes a live image viewer to appear to assist with animation development
-- `liveview::Bool`: Livestream the rendered animation to the local network
+- `livestreamto::Union{Symbol, Nothing}`: Livestream the rendered animation to the local network
+- `address::String` : Address for the livestream if streaming to local
+- `port::Int` : Port number for the livestream
+- `twitch_key::String` : The twitch streaming key for your account
```

Comment:
I was wondering whether we might want to have some kind of struct for it or a livestream function. Just to keep the number of parameters smaller but not sure yet.

Reply:
Yeah. That's needed for sure. We can have a constructor viz. `setup_livestream()` and pass the `struct` to the render function